<?xml version="1.0"?>
<launch>

<node pkg="foveated_stereo_ros" type="tracker_test" name="tracker_test" output="screen">
  <param name="output_frame" value="ego_frame"/>
  <param name="base_footprint_frame" value="base_footprint"/>
  <param name="egocentric_frame" value="ego_frame"/>
  <param name="freq" value="30.0"/>
  <param name="sensor_timeout" value="1.0"/>  
  <param name="odom_used" value="true"/>
  <param name="stereo_used" value="true"/>

  <remap from="stereo_data" to="stereo_data" />
</node>

</launch>
